content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1033,13 +1047,13 @@ by the cloud-controller-manager.
 
 ### batch.kubernetes.io/job-completion-index
 
-Type: Annotation
+Type: Annotation, Label
 
 Example: `batch.kubernetes.io/job-completion-index: "3"`
 
 Used on: Pod
 
-The Job controller in the kube-controller-manager sets this annotation for Pods
+The Job controller in the kube-controller-manager sets this as a label and annotation for Pods
```

Comment:
Why do we need the same information as both a label or annotation?

Reply:
See [KEP-4017](https://github.com/kubernetes/enhancements/pull/4019) for rationale behind this decision.